TestForSignUp/Registrator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace TestForSignUp
+{
+    internal static class Registrator
+    {
+        internal static bool RegistrateWithCorrectData(InputGenerator input)
+        {
+            try
+            {
+                var browser = new ChromeDriver();
+                browser.Navigate().GoToUrl("https://localhost:44335/signup");
+                Thread.Sleep(5000);
+                var filler = new ElementFiller(browser);
+                filler.Fill(input.commonString, input.emailString, input.dateString);
+                var buttons = browser.FindElementsByCssSelector("[type=\"button\"]");
+                buttons[1].Click();
+                Thread.Sleep(5000);
+                var submitButton = browser.FindElementByCssSelector("[type=\"Submit\"]");
+                submitButton.Click();
+                Thread.Sleep(5000);
+                var errors = browser.FindElementsByCssSelector("[style=\" color: red;\"]");
+                if (errors.Count == 1) return false;
+            }
+            catch (Exception)
+            {
+                return false;
+            }
+            return true;
+        }
+
+        internal static bool RegistrateWithIncorrectData()
+        {
+            var browser = new ChromeDriver();
+            browser.Navigate().GoToUrl("https://localhost:44335/signup");
+            Thread.Sleep(5000);
+            var filler = new ElementFiller(browser);
+            filler.Fill("A", "A", "1");
```

Comment:
How about fill it with sth like NotCorrectName - it is also will not pass the check but it will be more understandable
Also, I  don't understand where is incorrect birthday? 
And do you have to check all incorrect stuff in one test or it have to be for each error another one

Reply:
1, 2: fixed
3: I think we can check everything with one test.